Sources/Valet/Valet.swift

```diff
@@ -168,6 +179,9 @@ public final class Valet: NSObject, KeychainQueryConvertible {
     @objc(containsObjectForKey:)
```

Comment:
Might be worth noting in the header doc that it returns `false` when keychain is not accessible.